src/containers/AdminDashboard/AdminController.tsx

```diff
@@ -243,5 +245,84 @@ class AdminController {
       />
     );
   }
+
+  adminUserForm(): JSX.Element {
+    const { formError } = this.view.state;
+    return (
+      <div
+        className="material-content elevation3"
+        style={{ maxWidth: '320px', margin: '30px auto', padding: '10px 10px 20px 10px' }}
+      >
+        <h4 className="material-header-h4">
+          Add Admin User
+        </h4>
+        <form
+          id="modify-admins"
+          style={{
+            textAlign: 'left', marginLeft: '4px', width: '100%', maxWidth: '100%',
+          }}
+        >
+          <label htmlFor="addAdminEmail">
+            Admin Email
+            <input
+              id="addAdminEmail"
+              type="email"
+              placeholder="placeholder@gmail.com"
+              onChange={this.view.onChangeAdminEmail}
+            />
+          </label>
+          <input type="submit" disabled={this.validateAdmin()} onClick={this.addAdminUser} />
+          <p className="form-errors" style={{ color: 'red', marginBottom: '-15px' }}>{formError}</p>
+        </form>
+      </div>
+    );
+  }
+
+  validateAdmin(): boolean { // eslint-disable-line class-methods-use-this
+    let disabled = true,
+      validEmail = false;
+    const { addAdminEmail, formError } = this.view.state;
+    // eslint-disable-next-line no-useless-escape
+    const regEx = RegExp('^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$');
+    if (regEx.test(addAdminEmail) && addAdminEmail.includes('.') && addAdminEmail.includes('@gmail.com')) {
+      validEmail = true;
+    }
+    if (addAdminEmail !== '' && validEmail && !formError.includes('gmail')) disabled = false;
+    return disabled;
+  }
+
+  async addAdminUser(evt: { preventDefault: () => void; }): Promise<boolean | void> {
+    // Remove and move to API that controls validation
+    evt.preventDefault();
+    const { addAdminEmail } = this.view.state;
+    const userRoles: string[] = commonUtils.getUserRoles();
+    const { auth } = this.view.props;
+
+    // Alternatively can set checks to disable button permanently unless gmail account is added.
+    let r;
+    // determine if user exists or is already in the database
+    try {
+      r = await this.superagent.post(`${process.env.BackendUrl}/user`)
+        .set('Authorization', `Bearer ${auth.token}`)
+        .set('Accept', 'application/json')
+        .send({
+          email: addAdminEmail,
+        });
+    // eslint-disable-next-line no-console
+    } catch (e) { console.log(e); return false; }
+    if (r.status === 400) {
+      // Create user. Include userType for admin for clc (clc-admin)
+      this.view.setState({ formError: '' });
+      return true;
+    }
+    if (r.body._id && userRoles.indexOf(r.body.userType) === -1) {
+      // if user exists, and isn't an admin, update by user email. Set userType = clc-admin
+      return true;
+    }
+    // If user exists and is already a clc-admin, return error.
```

Comment:
again, please remove mention of userType here, also search for any other hard-coded userType and remove, those should only be referenced by the commonUtils.getUserType() or process.env.userRoles functions, thanks